### PR TITLE
Onboarding pr issue management

### PR DESCRIPTION
**Pull request by yimmee123**
## PR Checklist
- [x] Closes #xxx
- [x] Tests added/passed
- [x] Documentation updated
   - If checked, please file a pull request on [our docs repo](https://github.com/MicrosoftDocs/terminal) and link it here: #xxx
- [ ] Schema updated (if necessary)

### CODE REVIEW

File: .github/actions/spelling/allow/apis.txt [hunk 1]
```diff
@@ -29,6 +29,7 @@ Dacl
 dataobject
 dcomp
 DERR
+delayimp
```
Comment (yimmee123) [inline .github/actions/spelling/allow/apis.txt:32]:
```suggestion
DERR
```

Comment (yimmee123) [reply .github/actions/spelling/allow/apis.txt:32]:
```suggestion
DERR
```

**Comment by DHowett** [comment]
@yimmee123 you are not filing meaningful pull requests on this repository and I must ask you to stop.

**Comment by j4james** [comment]
@DHowett Accounts like this are almost certainly bots creating fake content to make their profiles seem more legitimate. They're used to sell github stars (see https://the-guild.dev/blog/judging-open-source-by-github-stars). When I'm feeling enthusiastic I report them as spam, and the GitHub mods will sometimes delete the accounts. Otherwise just block them.

**Pull request closed without merging**